.buildkite/pipeline.quick.yml

```diff
@@ -81,7 +81,7 @@ steps:
   - label: iOS 10 unit tests
     timeout_in_minutes: 10
     agents:
-      queue: opensource-mac-cocoa-11
+      queue: opensource-mac-cocoa-10.14
```

Comment:
Any particular need for 10.14 rather than 10.15?  We only have one 10.14 build server, but a few 10.15s.

Reply:
I tried 10.15 - those boxes don't have the iOS 10 simulator installed so the tests fail.

Reply:
OK, I'll get the simulators installed on them so we can bump the pipeline on at some point.